src/main/resources/templates/admin/ask/exchange.html

```diff
@@ -1,13 +1,19 @@
-<div class="question" th:text="${query}" th:if="${query} != null" />
-<div class="answer" th:utext="${answer}" th:if="${answer} != null" />
-<div class="error" th:text="${failureMessage}" th:if="${failureMessage} != null" />
+<div class="question" th:text="${query}" th:if="${query != null}" />
+<div class="answer" th:utext="${answer}" th:if="${answer != null}" />
+<div class="error" th:if="${answer == null && query != null}">
+    <b>Unable to get an answer!</b> This may be a temporary problem; wait a moment and try again.
+    Conversation ID:
+    <th:block th:utext="${conversationId}" />
+</div>
```

Comment:
You don't want to show the end user the failure message anymore?

Reply:
Yeah, it's just technical gobbeldygook from the point of view of the people who will be using this regularly.